text/0000-sends-accepts.md

```diff
@@ -0,0 +1,46 @@
+- Start Date: 2015-12-3
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This proposes adding `sends` and `accepts` attributes to the `Category` element
```

Comment:
I am not saying this particular change is good or bad, rather I am missing the reasoning why we think it is conceptually good to have mime types attached to the category element.